docs/strategy-callbacks.md

```diff
@@ -21,6 +22,31 @@ Currently available callbacks:
 !!! Tip "Callback calling sequence"
     You can find the callback calling sequence in [bot-basics](bot-basics.md#bot-execution-logic)
 
+## Bot start
+
+A simple callback which is called once when the bot starts.
+This can be used to perform actions that must only be performed once and runs after dataprovider and wallet are set
+
+``` python
+import asyncio
+
+class AwesomeStrategy(IStrategy):
+
+    # ... populate_* methods
+
+    async def some_asynchronous_task(self):
+        self.dp['remote_data'] = requests.get('https://some_remote_source.example.com')
+
+    def bot_start(self, **kwargs) -> None:
+        """
+        Called only once after bot instantiation.
+        :param **kwargs: Ensure to keep this here so updates to this won't break your strategy.
+        """
+        self.loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(self.loop)
+        self.loop.run_until_complete(self.some_asynchronous_task())
```

Comment:
This is a non-working example.

Please do NOT use asyncio for examples (especially not if they are faulty / non-working). The last line will block - so the bot will fail to start.

Therefore, please keep the example small and simple (for example do some fake-code to load some machine learning model).